src/js/components/FormField/FormField.js

```diff
@@ -287,19 +305,27 @@ const FormField = forwardRef(
     let abutMargin;
     let outerStyle = style;
 
-    if (themeBorder) {
-      const innerProps =
-        themeBorder.position === 'inner'
-          ? {
-              border: {
-                ...themeBorder,
-                side: themeBorder.side || 'bottom',
-                color: borderColor,
-              },
-              round: formFieldTheme.round,
-              focus,
-            }
-          : {};
+    if (themeBorder && themeBorder.position === 'inner') {
+      const innerProps = isFileInputComponent
```

Comment:
Can't comment on line 332, but now that we're not returning an empty object like we did when `themeBorder.position` !== `inner`, are we going to get any issues trying to spread `...innerProps` when the value is just `undefined` instead of `{}`? For example, if the theme border position was "outer" ?

Reply:
yes that was changing the `snap shots` I put it back and did the logic inline